File: Leetcode Js/L_Minimum_Number_Of_Moves_To_Seat_Everyone.js
// Minimum Number of Moves to Seat Everyone
// Array Sorting JavaScript

/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
var minMovesToSeat = function (seats, students) {
  seats.sort((a, b) => b - a);
  students.sort((a, b) => b - a);
  let res = 0;
  for (let k = 0; k < students.length; k++) {
    res += Math.abs(students[k] - seats[k]);
  }
  return res;
};

console.log(minMovesToSeat([3, 1, 5], [2, 7, 4]));
console.log(minMovesToSeat([4, 1, 5, 9], [1, 3, 2, 6]));
console.log(minMovesToSeat([2, 2, 6, 6], [1, 3, 2, 6]));

File: Leetcode/L_Minimum_Number_of_Moves_to_Seat_Everyone.py
# Minimum Number of Moves to Seat Everyone
# Array Sorting

class Solution:
    def minMovesToSeat(self, seats, students) -> int:
        seats.sort()
        students.sort()
        Total = 0
        for x, y in zip(seats, students):
            Difference = abs(x-y)
            Total = Total + Difference
        return Total


X = Solution()
print(X.minMovesToSeat([3, 1, 5], [2, 7, 4]))
print(X.minMovesToSeat([4, 1, 5, 9], [1, 3, 2, 6]))
print(X.minMovesToSeat([2, 2, 6, 6], [1, 3, 2, 6]))
